packages/appeal-reply-service-api/src/controllers/reply.js

```diff
@@ -75,25 +76,29 @@ module.exports = {
       .then(async (originalDoc) => {
         // logger.debug(`Original doc \n${originalDoc.reply}`);
         logger.debug(originalDoc.reply, 'STEVE_API');
-
-        const isFirstSubmission = originalDoc.state !== 'SUBMITTED' && newDoc.state === 'SUBMITTED';
-
-        if (isFirstSubmission) newDoc.submissionDate = new Date().toISOString();
-
         await mongodb
           .get()
           .collection(dbId)
-          .updateOne({ _id: idParam }, { $set: { uuid: idParam, reply: newDoc } })
-          .then(() => {
+          .updateOne({ _id: idParam }, { $set: { uuid: idParam, reply: req.body } })
+          .then(async () => {
             logger.debug(`Updated reply ${idParam}\n`);
 
-            if (isFirstSubmission) {
-              logger.info({ newDoc }, 'STEVE_IS_FIRST_SUBMISSION');
-              queue.addAppealReply(newDoc);
-              // TODO: call to notify will go here
-            }
+            // if (originalDoc.state !== 'SUBMITTED' && newDoc.state === 'SUBMITTED') {
+            //   logger.info({ newDoc }, 'STEVE_IS_FIRST_SUBMISSION');
+            //   await queue.addAppealReply(newDoc);
+            // }
```

Comment:
can be removed?

Reply:
No leaving it for steve just commented it out as its makiing build fail